app/frontend/src/features/connections/friends/useFriendList.test.tsx

```diff
@@ -0,0 +1,202 @@
+import { renderHook } from "@testing-library/react-hooks";
+import { QueryClient, QueryClientProvider } from "react-query";
+import React from "react";
+import { service } from "../../../service";
+import useFriendList from "./useFriendList";
+import { getUser, listFriends } from "../../../test/serviceMockDefaults";
+
+const listFriendsMock = service.api.listFriends as jest.Mock;
+const getUserMock = service.user.getUser as jest.Mock;
+
+const wrapper = ({ children }: { children: React.ReactNode }) => {
+  const client = new QueryClient({
```

Comment:
Inlining this in the wrapper is key to make sure a new client/cache is created for each test so they are isolated. This was causing me so much headaches before I realised!